res/skins/QMLDemo/EffectUnit.qml

```diff
@@ -1,75 +1,109 @@
 import "." as Skin
+import Mixxx 0.1 as Mixxx
 import QtQuick 2.12
 import QtQuick.Controls 2.12
 import QtQuick.Layouts 1.12
 import "Theme"
 
 Item {
```

Comment:
not sure if you think the effort is worth it, but you could add the expanded controls from latenight as well: 
![image](https://user-images.githubusercontent.com/12380386/122808134-be221c80-d2cc-11eb-98e1-a9e087123239.png)

Reply:
This PR is already quite big. Someone can take care of that separately I think. There's also other stuff missing, like the link type, or configuring the knob arc start from the parameter, etc.